relayer/src/listeners/azero.rs

```diff
@@ -135,50 +143,60 @@ impl AlephZeroListener {
 
         // Main AlephZero event loop
         loop {
-            // Query for the next unknown finalized block number, if not present we wait for it.
-            let mut to_block = get_next_finalized_block_number_azero(
-                azero_connection.clone(),
-                first_unprocessed_block_number,
-            )
-            .await;
-
-            to_block = min(to_block, first_unprocessed_block_number + (*sync_step) - 1);
-
-            info!(
-                "Processing events from blocks {} - {}",
-                first_unprocessed_block_number, to_block
-            );
-
-            // Add the next block numbers now, so that there is always some block number in the set.
-            for block_number in first_unprocessed_block_number..=to_block {
-                add_to_pending(block_number + 1, pending_blocks.clone()).await;
-            }
+            match emergency.load(Ordering::Relaxed) {
+                true => {
+                    trace!("Advisory is in an emergency state");
```

Comment:
I think this should be higher than `trace`, maybe `warn`, this is potentially important info

Reply:
warn will just flood your logs

Reply:
OK, that's a good idea, but nut sure if it won't get unnecessarily complex.
We could also sleep longer or make that duration configurable